### DISCUSSION TITLE
Didn't get education benefits after 3 days ,even id gets approved

### DISCUSSION THREAD

**Discussion by alshahriar-uu** [community/community#136745] [GitHub Education]
### Hi everyone,

Hi everyone,
i didn't get any education benefits even after my id gets approved already 3 days passed.
Is there anyway to get it quick soon?i badly need it urgent basis

**Comment by ygor-sena** [comment]
Hello @alshahriar-uu!

- Go to https://education.github.com/pack and see if you can log in to your account. Check if your benefits are listed there and if they're active;
- Otherwise, contact GitHub Support by writing a ticket if you still don't see the benefits: https://support.github.com/ 

I hope this information was useful. Best regards.

**Comment by davevad93** [comment]
Hi @alshahriar-uu ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @alshahriar-uu was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
